lib/hooks/usePrintOrder.ts

```
@@ -0,0 +1,45 @@
+
+export default function usePrintOrder() {
+  const configuration = useContext(ConfigurationContext);
+  const { printer, setPrinter } = useContext(Restaurant.Context);
+  const { loading, error, data } = useOrders();
+
+  const printOrder = async (id) => {
+    if (!loading && !error) {
+      const order = data.restaurantOrders.find(
+        (order: IOrder) => order._id === id
+      );
+      const result = await printAsync(
```

ESLint

[error] 'result' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)